Competition2017/src/competition/subsystems/drive/commands/DriveToPositionCommand.java

```diff
@@ -0,0 +1,48 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+
+import xbot.common.command.BaseCommand;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+import competition.subsystems.drive.DriveSubsystem;
+
+public class DriveToPositionCommand extends BaseCommand{
```

Comment:
It is 'To' an absolute distance though as opposed to a relative one. I can see your point about position vs distance but it this year sans mechanum we'll probably never have anything about x/y driving so probably won't get confused with anything else.